release/nightly_tests/dask_on_ray/large_scale_test.py

```diff
@@ -0,0 +1,456 @@
+"""
```

Comment:
Is this file currently used anywhere? Should it be added to the tests? (some test yaml?)

Reply:
I will add it after making sure shuffle test results are reported correctly. (in the followup PR)